docs/modules/install/pages/manual.adoc

```diff
@@ -2,23 +2,23 @@
 
 In order to develop on decidim, you'll need:
 
-* *Git* 2.15+
-* *PostgreSQL* 12.7+
-* *Ruby* 2.7.5
-* *NodeJS* 16.9.x
-* *Npm* 7.21.x
+* *Git* 2.34+
+* *PostgreSQL* 14.5+
+* *Ruby* 3.1.1
```

Comment:
For 0.27 we still have problem installing with Ruby 3.1 until #9849 is resolved.

I believe the particular issue that causes the installation to fail is already fixed at #9849 but I still haven't figured out how to fix the CI with those changes.

So I am thinking should we recommend Ruby 3.0.x for the time being or wait for #9849 to be resolved until bumping the version?

Reply:
My original idea was to have different versions of the installation guide with the different versions of Ruby for the different versions of Decidim. On that case, this would be the one for `develop`.

But in a second thought this is just a bad idea: we should change the `gem install decidim` to something else, and people just don't want to install the develop version with the gem command. 

So, I'll change it to what you say as that makes sense for sure.

Reply:
Done at acf818bb80758649205dcdc7135e836accdead0b